__main__.py

```
@@ -0,0 +1,607 @@
+    if args.output:
+        save_ctf_report(result, args.output)
+    else:
+        # Use timezone-aware timestamp in filename
+        save_ctf_report(
+            result,
+            f"ctf_report_{datetime.now().astimezone().strftime('%Y%m%d_%H%M%S%z')}.json"
+        )
+    
+    return result
+
+def run_targeted_ctf_test(bot: CyberShell, target: str, vuln_type: str):
```

Ruff (0.12.2)

`run_targeted_ctf_test` is too complex (12 > 10)

(C901)